src/libxrpl/protocol/Indexes.cpp

```diff
@@ -313,6 +315,21 @@ depositPreauth(AccountID const& owner, AccountID const& preauthorized) noexcept
         indexHash(LedgerNameSpace::DEPOSIT_PREAUTH, owner, preauthorized)};
 }
 
+Keylet
+depositPreauth(
```

Comment:
Please add a comment that the hash requires the list be sorted (for future reference).

Reply:
I suppose you look at the old version, there is no list.

Reply:
Sorry, I meant set. I know a set is effectively automatically sorted, but I just wanted to leave a comment in case anyone thinks about refactoring this code in the future.